processes/local_development/seed_test_data.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+from uuid import uuid4
+from mappings.base_mapping import BaseMapping
+from datetime import datetime, timezone
+from model import Record
+from constants.get_constants import get_constants
+from logger import create_log
+from managers import RedisManager
+from processes import CatalogProcess, ClassifyProcess, ClusterProcess, HathiTrustProcess
+
+logger = create_log(__name__)
+
+class TestMapping(BaseMapping):
+    def __init__(self, record):
+        self.record = record
+
+    def createMapping(self):
+         pass
+
+class SeedTestDataProcess():
+    def __init__(self, *args):
+        super(SeedTestDataProcess, self).__init__(*args[:4])
+
+        self.constants = get_constants()
```

Comment:
not quite sure if we are going to using self.constants but let's remove it for now